adapters/repos/db/migrator.go

```diff
@@ -81,6 +81,17 @@ func (m *Migrator) AddClass(ctx context.Context, class *models.Class,
 			}
 		}
 
+		if class.InvertedIndexConfig.IndexPropertyLength {
+			dt := schema.DataType(prop.DataType[0])
+			// some datatypes are not added to the inverted index, so we can skip them here
+			if !(dt == schema.DataTypeGeoCoordinates || dt == schema.DataTypePhoneNumber || dt == schema.DataTypeBlob ||
+				dt == schema.DataTypeInt || dt == schema.DataTypeNumber || dt == schema.DataTypeBoolean || dt == schema.DataTypeDate) {
+				err = idx.addPropertyLength(ctx, prop)
+				if err != nil {
+					return errors.Wrapf(err, "extend idx '%s' with property length", idx.ID())
+				}
+			}
```

Comment:
Maybe a switch statement would be more readable here...

Reply:
also done, thx